kvbc/test/categorization/updates_test.cpp

```diff
@@ -23,10 +24,131 @@ TEST(categorized_updates, merkle_update) {
   ASSERT_TRUE(mu.getData().kv.size() == 1);
 }
 
-TEST(categorized_updates, add_block) {
-  KeyValueBlockchain block_chain;
-  Updates updates;
-  ASSERT_TRUE(block_chain.addBlock(updates) == (BlockId)8);
+TEST(categorized_updates, add_blocks) {
+  auto native_client = NativeClient::newClient("/tmp/add_block", false, NativeClient::DefaultOptions{});
```

Comment:
The storage_test_common.h file contains native client handling and cleanup. Can be reused here too.